libkbfs/config_local.go

```diff
@@ -55,6 +55,12 @@ const (
 	// By default, this will be the block type given to all blocks
 	// that aren't explicitly some other type.
 	defaultBlockTypeDefault = keybase1.BlockType_DATA
+
+	// By default, allow 10% of the free bytes on disk to be used in the disk block cache.
+	defaultDiskBlockCacheFraction = 0.10
+
+	// By default, allow 15% of the free bytes on disk to be used in the sync block cache.
+	defaultSyncBlockCacheFraction = 0.15
```

Comment:
Looks like I was wrong about the sync cache size being 15% before.  There seems to be some reporting bug related to my use of the sync cache, that makes the limit different from the working set cache limit.  For now, I say we keep this at `0.10` (change the param default too), and we can change it later if needed.